sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandler.java

```diff
@@ -475,7 +479,7 @@ private void emitHtmlBody(Map<String, String> queryMap, PrintStream out)
     String spanName = queryMap.get(PARAM_SPAN_NAME);
     if (spanName != null) {
       // Convert spanName with URL encoding
-      spanName = URLEncoder.encode(spanName, "UTF-8");
```

Comment:
It's the "job" of parseQueryMap to do URL encoding, not this business logic do it there. This is important

- if we add more parameters
- if someone uses this in their own server instead of the default sun http

Reply:
Ah yeah, that encode is redundant.

Reply:
Oh wait, the code didn't update (?). The encode should be replaced with a decode.

Reply:
Sorry I accidentally commented on the wrong line. I meant the line below - moving decode to parseQueryMap with related unit tests